CHANGES.md

```diff
@@ -1,6 +1,12 @@
 ringpop-go changes
 ==================
 
+v0.x.y (to be released)
```

Comment:
I think whatever works as long as we don't have a fixed number. I used "unreleased" before.